redbeams/src/main/java/com/sequenceiq/redbeams/flow/stack/provision/action/StackProvisionActions.java

```diff
@@ -0,0 +1,194 @@
+package com.sequenceiq.redbeams.flow.stack.provision.action;
+
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class StackProvisionActions {
```

Comment:
Though a cf stack, I still would rename it to DatabaseServerProvisionActions. 
In order to avoid adding this many more times: I would rename everything that has StackProvision in its name:
StackProvision => DatabaseServerProvision